lte/gateway/python/integ_tests/s1aptests/test_multi_enb_multi_ue.py

```diff
@@ -0,0 +1,94 @@
+"""
+Copyright (c) 2016-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+"""
+import time
+import unittest
+import s1ap_types
+import s1ap_wrapper
+
+
+class TestMultiEnbMultiUEAttachDetach(unittest.TestCase):
```

Comment:
The file name can reflect the test name, i.e. test_multi_enb_multi_ue_attach_detach.py